Modules/socketmodule.c

```diff
@@ -5574,6 +5574,29 @@ static PyType_Spec sock_spec = {
     .slots = sock_slots,
 };
 
+static PyObject *
+socket_getattr(PyObject *self, PyObject *name)
+{
+    PyObject *sock_type = PyType_FromSpec(&sock_spec);
+    if (sock_type == NULL) {
+        return NULL;
+    }
+    const char *attrname = PyUnicode_AsUTF8(name);
+    if (attrname == NULL) {
+        Py_DECREF(sock_type);
+        return NULL;
+    }
+
+    if (!strcmp(attrname, "SocketType")) {
```

Comment:
I'm not sure where my other review comment went, but again, please use `PyUnicode_CompareWithASCIIString` instead of manually calling `PyUnicode_AsUTF8` and then `strcmp`ing.

Reply:
Or you can use the nice PyUnicode_EqualToUTF8() function.

Reply:
Oh, that's new in 3.13, TIL!